pkg/efivarfs/vars.go

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// Package efivarfs allows interaction with efivarfs of the
+// linux kernel.
+
```

Comment:
No empty line for package comments.

Reply:
Package comments *must* be immediately preceding the package statement.

Reply:
awesome sauce, and done